ftw/raven/config.py

```diff
@@ -32,6 +33,36 @@ def ignored_exception_classnames(self):
 
         return tuple(ignored)
 
+    @property
+    def tags(self):
+        value = self._get_tags_from_file()
+        value.update(self._get_tags_from_env())
+        return value
+
+    def _get_tags_from_env(self):
+        value = self._get_stripped_env_variable('RAVEN_TAGS')
+        if not value:
+            return {}
+
+        try:
+            return json.loads(value)
+        except ValueError:
+            return {'RAVEN_TAGS': 'invalid configuration'}
+
+    def _get_tags_from_file(self):
+        value = self._get_stripped_env_variable('RAVEN_TAGS_FILE')
+        if not value:
+            return {}
+
+        if not os.path.exists(value):
```

Comment:
Better: `os.path.isfile()` (`os.path.exists()` will also return `True` for directories).